### PR TITLE
AB#41737 Move UserSettings table into User table

### PR DESCRIPTION
**Pull request by jamesgriff**
There is currently a table called UserSettings, which should really be a bunch of columns on the User table.
Storing the data as a separate table causes various problems because the values are stored as strings instead of the real data types (e.g. DateTimes)
This change does that migration.

### CODE REVIEW

File: GenderPayGap.Database.Models/Extensions/User.cs [hunk 1]
```diff
@@ -150,58 +120,6 @@ public void SetStatus(UserStatuses status, User byUser, string details = null)
...
-            return verifyUrl;
-        }
-
-        public string GetPasswordResetUrl()
-        {
-            string resetCode = Encryption.EncryptQuerystring(UserId + ":" + VirtualDateTime.Now.ToSmallDateTime());
-            string resetUrl = $"/register/enter-new-password?code={resetCode}";
-            return resetUrl;
-        }
-
```
Comment (jamesgriff) [inline GenderPayGap.Database.Models/Extensions/User.cs:204]: These two methods are just some unrelated unused code

File: GenderPayGap.Database/GenderPayGap.Database.csproj [hunk 2]
```diff
@@ -46,5 +46,10 @@
 
     <!-- Include the layer model as an additional file that will be passed to the analyzer -->
   </ItemGroup>
+  <ItemGroup>
+    <None Update="appsettings.json">
```
Comment (raamSoftwire) [inline GenderPayGap.Database/GenderPayGap.Database.csproj:50]: Could you explain what is happening with this file please?

File: GenderPayGap.WebUI/Controllers/Account/AccountCreationController.cs [hunk 3]
```diff
@@ -177,8 +177,8 @@ private User CreateNewUser(CreateUserAccountViewModel viewModel)
             user.Lastname = viewModel.LastName;
             user.JobTitle = viewModel.JobTitle;
             user.EmailAddress = viewModel.EmailAddress;
-            user.SetSetting(UserSettingKeys.AllowContact, viewModel.AllowContact.ToString());
-            user.SetSetting(UserSettingKeys.SendUpdates, viewModel.SendUpdates.ToString());
+            user.AllowContact = viewModel.AllowContact;
+            user.SendUpdates = viewModel.SendUpdates;
```
Comment (raamSoftwire) [inline GenderPayGap.WebUI/Controllers/Account/AccountCreationController.cs:181]: This is so much nicer now!
